interface/main/calendar/includes/pnMod.php

```diff
@@ -700,6 +700,8 @@ function pnModURL($modname, $type = 'user', $func = 'main', $args = array(), $pa
     //this may break windows path accesses?
     $path = preg_replace("/[^\.\/a-zA-Z0-9]/", "", $path)
     ;
+    //Escapes several special chars
+    $url=attr($url);
```

Comment:
hi @matrix-amiel ,
Note I had to revert this in the main codebase, since it caused the following bug:
https://github.com/openemr/openemr/issues/1105
-brady